config/templates/-icon-acquia-rocket.hbs

```diff
@@ -0,0 +1,12 @@
+                                  <path d="M12.5287288,9.60947368 C12.7176554,9.99684211 12.4900847,10.5094737 12.0719774,10.6373684 C11.6871469,10.7810526 11.2132203,10.5636842 11.0833333,10.18 C10.9239266,9.79421053 11.1579379,9.31421053 11.5626271,9.19105263 C11.9302825,9.05684211 12.3838136,9.24947368 12.5287288,9.60947368 Z" id="Fill-7"></path>
+                              </g>
+                          </g>
+                      </g>
+                  </g>
+              </svg>
```

Comment:
Just me being oblivious as usual - where did all the new files and hbs come from?

Reply:
no, as usual, you're right we didn't need those files